unit-tests/func/hw-errors/l500/l500-error-common.h

```diff
@@ -55,45 +55,41 @@ void trigger_error_or_exit( const rs2::device & dev, uint8_t opcode, uint8_t num
     }
 }
 
+// This function loop over all posiable hw errors use hw-monitor command to trigger them
+// and check that we call to user callback with the error
 void validate_errors_handling( const rs2::device & dev,
```

Comment:
Is this function specific to L500? I'm wondering why it's in this header

Reply:
moved to error-common.h